examples/jsm/csm/CSMFrustum.js

```diff
@@ -8,6 +8,8 @@ class CSMFrustum {
 
 		data = data || {};
 
+		this.zNear = data.webGL === true ? - 1 : 0;
```

Comment:
This module creates internal instances of `CSMFrustum` but without honoring the `data` parameter (see line 87).

It seems `data` must be a property so you can pass it later to the ctor.

Reply:
the webGL parameter is only used in setFromProjectionMatrix() which is only ever called for the mainFrustum().  The temporary  _lightSpaceFrustum and the frustums generated by mainFrustum.split() never have this method called and inherit their vertices from the mainFrustum AFAICS (in CSMShadowNode and the original implementation).

Also the CSMFrustum is never created with a data.projectionMatrix value, so 
https://github.com/mrdoob/three.js/blob/37d6f280a5cd642e801469bb048f52300d31258e/examples/jsm/csm/CSMFrustum.js#L26-L30
seems redundant.